pkg/metrics/metrics.go

```diff
@@ -86,7 +86,7 @@ func Validate(ctx context.Context, cfg *rest.Config, namespace string) error {
 
 	// Check if metrics service exists
 	serviceList, err := oclient.CoreV1().Services(namespace).List(ctx,
-		metav1.ListOptions{LabelSelector: "name=windows-machine-config-operator"})
+		metav1.ListOptions{LabelSelector: "name=windows-exporter"})
```

Comment:
Use a variable/constant here to make it clear whats going on. The comment at definition of the constant should mention that the name is set in the service manifest yaml